public/app/features/explore/state/main.ts

```diff
@@ -105,23 +122,16 @@ export function splitOpen<T extends DataQuery = any>(options?: {
         };
       }
 
-      dispatch(splitOpenAction({ itemState: rightState }));
-
-      const queries = [
+      rightState.queries = [
         {
           ...options.query,
           refId: 'A',
         } as DataQuery,
       ];
 
-      const dataSourceSettings = getDatasourceSrv().getInstanceSettings(options.datasourceUid);
-
-      await dispatch(changeDatasource(ExploreId.right, dataSourceSettings!.name));
-      await dispatch(setQueriesAction({ exploreId: ExploreId.right, queries }));
-      await dispatch(runQueries(ExploreId.right));
+      dispatch(splitOpenAction({ itemState: rightState }));
```

Comment:
When opening new split pane we count on ExplorePaneContainer to do it's job and initialize state properly so we should not be doing that 2 times.